frontend/src/app/survey/detail/result-overview/result-overview.component.ts

```diff
@@ -0,0 +1,44 @@
+import {Component, Input, OnInit} from '@angular/core';
+import {TimeslotDto} from '../../../api/models/timeslot-dto';
+import {ResponseDto} from '../../../api/models/response-dto';
+
+@Component({
+    selector: 'app-result-overview',
+    templateUrl: './result-overview.component.html',
+    styleUrls: ['./result-overview.component.sass']
+})
+export class ResultOverviewComponent implements OnInit {
+
+    @Input() timeSlots: TimeslotDto[] = [];
+    @Input() responses: ResponseDto[] = [];
+    displayedColumns: string[] = [];
+
+    constructor() {
+    }
+
+    ngOnInit(): void {
+        setTimeout(() => {
+            this.displayedColumns = ['name', ...this.timeSlots.map(timeslot => timeslot.id.toString())];
+        }, 500);
```

Comment:
Why do you add a delay here?

Reply:
Don't ask :P There are problems in angular with async component rendering. This was the only solution which worked. Searching for a cleaner approach might need further reseacher